### ISSUE TITLE
PEP 440 compliance

### ISSUE THREAD

**Issue by dsuch** [akesterson/dpath-python#23]
Hi there,

I have been using dpath for several months and it's all good, nice work.

However, one thing that recently changed in setuptools is that it looks it apparently treats differently versions not compatible with PEP 440

In particular, when using setuptools under zc.buildout, when once wants to install, say, dpath 1.2-70 setuptools turns the version into 1.2.post70 and it ends with

```
Error: There is a version conflict.
We already have: dpath 1.2.post70
```

This is part of a bigger build mechanism so I'm not sure exactly which part of setuptools or zc.buildout does it but the root cause is that this is 1.2-70 rather than 1.2.70.

The question is, could you please start using the latter format for package versions? It doesn't make any real difference except that it makes both setuptools and zc.buildout happy.

Thanks.

**Comment by akesterson** [comment]
I have no objections to this, we'll get around to it soon-ish. Hopefully within a week or two.

**Comment by msabramo** [comment]
See https://github.com/akesterson/dpath-python/pull/29

**Comment by akesterson** [comment]
This is fixed with 1.3.2, thanks @msabramo for the patch.
